src/followButton.js

```diff
@@ -0,0 +1,16 @@
+const followButton = document.getElementById("profile-content-follow-button")
+
+const followButtonHelper = () => {
+    var currentText = document.getElementById("profile-content-follow-button");
+
+    if(currentText.innerHTML == "Follow") {
+        currentText.innerHTML = "Unfollow";
```

Comment:
Single Responsibility Principle
This function is doing three things right now:
1. Retrieving and comparing the current text of the follow button
2. Does the handling for unfollowing
3. Does the handling for following

Ideally it should only do the first one